lib/data_update_scripts/20201015190914_update_article_main_image_path.rb

```diff
@@ -0,0 +1,15 @@
+module DataUpdateScripts
+  class UpdateArticleMainImagePath
```

Comment:
This only updates `article.main_image` on Forem Cloud's apps. This updates the proxy path and removes the Cloudinary prefix.